src/odemis/driver/test/tescan_test.py

```diff
@@ -266,13 +266,17 @@ def test_light(self):
         """
         light = self.light
         orig_pwr = light.power.value[0]
-        new_pwr = int(not orig_pwr)  # 0 or 1
-        light.power.value[0] = new_pwr
-        self.assertEqual(light.power.value[0], new_pwr)
+        if orig_pwr == light.power.range[0]:
```

Comment:
I've actually turned it upside-down and changed `orig_pwr`.
This code is a little tricky, because we don't have a simulator... so it's not easy to run the code.